SDKGenerator.njsproj

```diff
@@ -186,17 +188,50 @@
     <Content Include="targets\csharp\source\PlayFabSDK\source\Uunit\UUnitTestReport.cs" />
     <Content Include="targets\csharp\source\PlayFabSDK\source\Uunit\UUnitTestSuite.cs" />
     <Content Include="targets\csharp\source\PlayFabSDK\source\WsaReflectionExtensions.cs" />
+    <Content Include="targets\csharp\source\PlayFabSDK\UnittestRunner\App.config" />
+    <Content Include="targets\csharp\source\PlayFabSDK\UnittestRunner\Properties\AssemblyInfo.cs" />
+    <Content Include="targets\csharp\source\PlayFabSDK\UnittestRunner\UnittestRunner.csproj" />
+    <Content Include="targets\csharp\source\PlayFabSDK\UnittestRunner\UUnitTestRunner.cs" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\App.config" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\bin\Debug\PlayFabAllSDK.dll" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\bin\Debug\PlayFabAllSDK.pdb" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\bin\Debug\WindowsFormsApp1.exe" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\bin\Debug\WindowsFormsApp1.exe.config" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\bin\Debug\WindowsFormsApp1.pdb" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\Form1.cs" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\Form1.Designer.cs" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\Form1.resx" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\obj\Debug\build.force" />
+    <Content Include="targets\csharp\source\PlayFabSDK\WindowsFormsApp1\obj\Debug\DesignTimeResolveAssemblyReferences.cache" />
```

Comment:
are all these obj directories necessary?

Reply:
probably not...
(thanks)